ios/Podfile.lock

```diff
@@ -1852,7 +1852,7 @@ PODS:
   - RNGoogleSignin (10.0.1):
     - GoogleSignIn (~> 7.0)
     - React-Core
-  - RNLiveMarkdown (0.1.70):
```

Comment:
why do we have the Podfile.lock changes in this one?

Reply:
Missed to restore this change when running IOS.

Reply:
Reverted this change.